src/main/java/org/broadinstitute/hellbender/utils/minimap2/MiniMap2Alignment.java

```diff
@@ -0,0 +1,57 @@
+package org.broadinstitute.hellbender.utils.minimap2;
+
+import java.nio.ByteBuffer;
+import java.util.Objects;
+
+public class MiniMap2Alignment {
+    private final int refId;
+    private final int refStart;
+    private final boolean revStrand;
+    private final int mapQ;
+    private final String cigar;
+
+    public MiniMap2Alignment( final ByteBuffer alignBuffer ) {
+        refId = alignBuffer.getInt();
+        refStart = alignBuffer.getInt();
+        final int mapQAndStrand = alignBuffer.getInt();
+        if ( mapQAndStrand < 0 ) {
+            revStrand = true;
+            mapQ = ~mapQAndStrand;
+        } else {
+            revStrand = false;
+            mapQ = mapQAndStrand;
+        }
+
+        final StringBuilder sb = new StringBuilder();
+        int nCigarWords = alignBuffer.getInt();
+        while ( nCigarWords-- > 0 ) {
+            final int cigarWord = alignBuffer.getInt();
+            final char operator = "MIDNSH".charAt(cigarWord & 0x0f);
```

Comment:
Would it be sensible to change this to support =/X operators? It looks like minimap 2 can emit them if someone calls setFlags(MiniMap2Aligner.MM_F_EQX).
```suggestion
            final char operator = "MIDNSHP=X".charAt(cigarWord & 0x0f);
```

Reply:
Yes.  Good thought.